app/api/groups/get_permissions_can_request_help_to.feature

```diff
@@ -55,5 +69,124 @@ Feature: Get permissions can_request_help_to for a group
+      | @Item | @ClassParent             | @OtherSourceGroup | other            | false    | @HelperOtherSourceGroup4          | other source group                                |
+      | @Item | @ClassParentParent       | @OtherSourceGroup | other            | false    | @HelperOtherSourceGroupNotVisible | other source group, not visible                   |
+  # The following lines are to make the groups visible by @Teacher
+  And the group @Teacher is a descendant of the group @HelperGroupSelf1
+  And the group @Teacher is a descendant of the group @HelperGroupGroupMembership1
+  And the group @Teacher is a descendant of the group @HelperGroupGroupMembership2
+  And the group @Teacher is a descendant of the group @HelperGroupItemUnlocking
+  And the group @Teacher is a descendant of the group @HelperGroupOther1
+  And the group @Teacher is a descendant of the group @HelperGroupOther2
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup1
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup2
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup3
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup4
+  When I send a GET request to "/groups/@Class/permissions/@Class/@Item"
+  Then the response code should be 200
+  And the response at $.granted_via_self.can_request_help_to[*] should be:
+    | id                       | name                          | is_all_users_group |
+    | @AllUsers                | AllUsers                      | true               |
+    | @HelperGroupSelf1        | Group HelperGroupSelf1        | false              |
+    | @HelperOtherSourceGroup1 | Group HelperOtherSourceGroup1 | false              |
+  And the response at $.granted.can_request_help_to should be:
+    | id                           | name                              |
+    | @HelperGroupGroupMembership1 | Group HelperGroupGroupMembership1 |
+  And the response at $.granted_via_group_membership.can_request_help_to[*] should be:
+    | id                           | name                              | is_all_users_group |
+    | @HelperGroupGroupMembership2 | Group HelperGroupGroupMembership2 | false              |
+    | @AllUsers                    | AllUsers                          | true               |
+    | @HelperOtherSourceGroup2     | Group HelperOtherSourceGroup2     | false              |
+  And the response at $.granted_via_item_unlocking.can_request_help_to[*] should be:
+    | id                        | name                           | is_all_users_group |
+    | @HelperGroupItemUnlocking | Group HelperGroupItemUnlocking | false              |
+    | @AllUsers                 | AllUsers                       | true               |
+    | @HelperGroupNotVisible    |                                | false              |
```

Comment:
This `name` value does not really validate if it is a omitted value, null or "". I would say it is is not just a detail as typically bugs hide in these kind of things.

Reply:
fixed, see comment below for how it is checked